go.mod

```
@@ -220,3 +220,6 @@ require (
 	gopkg.in/ini.v1 v1.66.4 // indirect
 	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
 )
+
+// replace github.com/minio/console => ../console
+replace github.com/minio/console => github.com/donatello/console v0.12.6-0.20220713224721-6753ce0162cb
```

```suggestion
replace github.com/minio/console => github.com/donatello/console v0.12.6-0.20220713224721-6753ce0162cb
```

This can be moved to `console` master @donatello